src/main/java/ru/svgogin/service/sparkservice/SparkServiceApplication.java

```diff
@@ -0,0 +1,13 @@
+package ru.svgogin.service.sparkservice;
```

Comment:
I think, it will be better to change package name.
`service` occurs twice in package name.
You can change to:
```suggestion
package ru.svgogin.service.spark;
```
or 
```suggestion
package ru.svgogin.spark;
```
It also requires to change your artifactId in pom.xml